exasol_data_science_utils_python/model_utils/udfs/partial_fit_regression_train_udf.py

```diff
@@ -24,7 +25,7 @@ def run(self, exa, ctx, model, table_preprocessor_factory: TablePreprocessorFact
         source_schema = SchemaName(ctx.source_schema_name)
         target_schema = SchemaName(ctx.target_schema_name)
         experiment_name = ExperimentName(ctx.experiment_name)
-        source_table = TableName(ctx.source_table_name, schema=source_schema)
+        source_table = TableNameBuilder.create(ctx.source_table_name, schema=source_schema)
         columns = [ColumnName(column_name, table_name=source_table)
                          for column_name in ctx.columns.split(",")]
```

Comment:
here you are not using the ColumnNameBuilder, is this correct?

Reply:
done